kapitan/cli.py

```diff
@@ -286,13 +284,25 @@ def build_parser():
         default=from_dot_kapitan("compile", "yaml-multiline-string-style", "double-quotes"),
         help="set multiline string style to STYLE, default is 'double-quotes'",
     )
-
     compile_parser.add_argument(
         "--yaml-dump-null-as-empty",
         default=from_dot_kapitan("compile", "yaml-dump-null-as-empty", False),
         action="store_true",
         help="dumps all none-type entries as empty, default is dumping as 'null'",
     )
+    compile_parser.add_argument(
+        "--helm-secrets",
+        action="store_true",
+        default=from_dot_kapitan("compile", "helm-secrets", False),
+        help="enable kapitan secret engine on helm refs",
+    )
+    compile_parser.add_argument(
+        "--encode-base64",
```

Comment:
Since this is helm specific, maybe call it instead `--helm-refs-base64`?

Reply:
Done in the latest commit